models/Product.js

```diff
@@ -1,2 +1,64 @@
 'use strict';
 
+const sqlite3 = require('sqlite3').verbose();
+const db = new sqlite3.Database('./db/bangazon.sqlite');
+
+module.exports.dbGetAllProducts = () => {
+  return new Promise( (resolve, reject) => {
+    db.all(`SELECT * FROM products`, (err, productdata) => {
+      if(err) reject(err);
+      resolve(productdata);
+    });
+  });
+};
+
+module.exports.dbGetSingleProduct = (id) => {
+  return new Promise( (resolve, reject) => {
+    db.get(`SELECT * FROM products
+            WHERE id = ${id}`, (err, productdata) => {
+      if(err) reject(err);
+      resolve(productdata);
+    });
+  });
+};
+
+module.exports.dbPostProduct = (newProduct) => {
+  return new Promise((resolve, reject) => {
+      let { type_id, price, title, description, original_quantity, seller_user_id } = newProduct;
+      db.run(`INSERT INTO products(product_type_id, price, title, description, original_quantity, seller_user_id)
+      VALUES('${type_id}', '${price}', '${title}', '${description}', '${original_quantity}', '${seller_user_id}')`, (err) => {
```

Comment:
'type_id' should be 'product_type_id'
this is putting undefined into product type as is.